TimedArithmeticQuiz2/src/models/QuestionModel.js

```diff
@@ -0,0 +1,38 @@
+import newId from "../utility/newId"
```

Comment:
Did anyone suggested the folder structure? We don't really use the model structure on the FE.

Reply:
no not suggested by anyone. will remove default export/import.